ick/runner.py

```diff
@@ -102,13 +143,17 @@ def _perform_test(self, rule_instance, test_path) -> None:
             bp = test_path / "b"
             files_to_check = set(glob("*", root_dir=bp, recursive=True))
             files_to_check.update(glob(".github/**", root_dir=bp, recursive=True))
+            files_to_check = {f for f in files_to_check if (bp / f).is_file()}
 
             response = self._run_one(rule_instance, repo, project)
-            assert isinstance(response[-1], Finished), "Last response is finished"
+            if not isinstance(response[-1], Finished):
+                raise AssertionError(f"Last response is not Finished: {response[-1].__class__.__name__}")
             if response[-1].error:
```

Comment:
This check means we don't need the assert on line 165, right?

Reply:
yes, 100% line 165 is wrong now